news/management/commands/process_donations_queue.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import print_function, unicode_literals
+
+import json
+import sys
+from time import time
+
+from django.conf import settings
+from django.core.management import BaseCommand, CommandError
+
+import boto3
+import requests
+from django_statsd.clients import statsd
+
+from news.tasks import process_donation
+
+
+class Command(BaseCommand):
+    snitch_delay = 300  # 5 min
+    snitch_last_timestamp = 0
+    snitch_id = settings.DONATE_SNITCH_ID
+
+    def snitch(self):
+        if not self.snitch_id:
+            return
+
+        time_since = int(time() - self.snitch_last_timestamp)
+        if time_since > self.snitch_delay:
+            requests.post('https://nosnch.in/{}'.format(self.snitch_id))
```

Comment:
Are you throttling to play nice with DMS or is there another reason?

This is a nice idea for [babis](https://github.com/glogiotatidis/babis) btw, thanks!